package.json

```diff
@@ -24,5 +24,8 @@
   "version": "1.6.0-git",
   "bugs": {
     "url": "https://github.com/radareorg/radare2/issues"
+  },
+  "dependencies": {
+    "r2r": "0.0.0-beta.0"
```

Comment:
Why?

Reply:
And why did you add a package-lock.json file? Please remove both those changes

Reply:
Removed this change